### ISSUE TITLE
"Parameter indices which did not receive grad" When I Prompt-Tuned YOLO World

### ISSUE THREAD

**Issue by ZYJIQVV** [AILab-CVC/YOLO-World#523]
Dear Team,
I tried to finetune YOLO-World model with prompt tuning using the provided config file ```configs/prompt_tuning_coco/yolo_world_v2_l_vlpan_bn_2e-4_80e_8gpus_prompt_tuning_coco.py``` on MS COCO dataset. "Other than" the datast path and pretrained checkpoint path I did not modify the config file. But I encountered the error that some of the model parameters did not receive gradients in the neck.
Details are displayed below:
![image](https://github.com/user-attachments/assets/9e1c31f4-85c6-4e11-a317-bde9fd4b03d0)

I also tried to finetune with another config file ```configs/prompt_tuning_coco/yolo_world_v2_l_vlpan_bn_2e-4_80e_8gpus_mask-refine_prompt_tuning_coco.py``` and there was not such a problem.

The reason I use the config file without ```mask-refine``` is there is no mask in my COCO-formatted dataset.

COuld you please remind me what the problem could be can how I can fix this problem?

**Comment by nimeidi** [comment]
@ZYJIQVV Do you fix this problem?

**Comment by nimeidi** [comment]
> [@ZYJIQVV](https://github.com/ZYJIQVV) Do you fix this problem?

add "find_unused_parameters=True" in config file solve my problem
